packages/preview/delegis/0.1.0/template/main.typ

```diff
@@ -0,0 +1,56 @@
+#import "@preview/delegis:0.1.0": *
+
+#show: it => delegis(
+  // Metadata
+  title: "Vereinsordnung zu ABCDEF",
+  abbrev: "ABCDEFVO",
+  resolution: "3. Beschluss des Vorstands vom 24.01.2024",
+  inEffect: "24.01.2024",
```

Comment:
while not strictly required, I would suggest using `kebab-case` for variable and argument names as it is the idiomatic Typst case convention

Reply:
Ah, true. I'll fix that (I worked too much in other languages this week, still working on getting back into the Typst way of things).